lib/src/command/run.dart

```diff
@@ -22,13 +24,20 @@ class RunCommand extends PubCommand {
   RunCommand() {
     argParser.addFlag("checked",
         abbr: "c", help: "Enable runtime type checks and assertions.");
+    argParser.addFlag('list',
```

Comment:
To me once this moves to the `deps` command then `--list` is no longer descriptive enough (I would expect it to just list deps not executables). Maybe `--list-executables`?